category_theory/const.lean

```diff
@@ -47,4 +48,13 @@ include 𝒟
 
 end
 
+omit 𝒥
+def of : C ⥤ (punit ⥤ C) := const punit
```

Comment:
Could we have a docstring on `of`, stating what it does? I think it would also be good to put an explicit universe annotation on `punit` here, so it's clear that it is independent of the universe levels for `C`.

Reply:
Done.